canary/gradle

```diff
@@ -0,0 +1 @@
+../gradle
```

Comment:
I remember it cause some problem, maybe related to docker

Reply:
This is causing windows failing. We can duplicate the folder if needed